pkg/config/constants/constants.go

```diff
@@ -60,8 +60,14 @@ const (
 	IstioIngressNamespace = "istio-system"
 
 	// DefaultKubernetesDomain the default service domain suffix for Kubernetes, if not overridden in config.
+	// TODO(nmittler): Rename this to DefaultClusterLocalDomain.
+	// TODO(nmittler): Search/replace explicit usages of the string with this constant.
 	DefaultKubernetesDomain = "cluster.local"
 
+	// DefaultClusterSetLocalDomain is the default domain suffix for Kubernetes Multi-Cluster Services (MCS)
+	// used for load balancing requests against endpoints across the ClusterSet (i.e. mesh).
+	DefaultClusterSetLocalDomain = "clusterset.local"
```

Comment:
very nit/question: is it *default* or is it always clusterset.local in the MCS spec?

Reply:
Always via the spec, but you can always provide an alias.

On a related note, I think we should technically be supporting `cluster.local` even when the user specifies `domainSuffix`.  We should follow this up with a bit of a re-think on `domainSuffix` as a whole.